code/API_definitions/EdgeCloud_LcM.yaml

```diff
@@ -0,0 +1,658 @@
+                  - RESTART_POLICY_ALWAYS
+                  - RESTART_POLICY_NEVER
+                description: How the platform shall handle component failure                        
+      required:
+        - name
+        - virtType
+        - fileName
+        - repository
+        - componentSpec
+    CpuArchType:
+          type: string
+          enum:
+            - x86_64
+            - arm_64
+          description: CPU Instruction Set Architecture (ISA) E.g., Intel, Arm etc. 
+    ProvisionedApp:
+      description: Information about the provisioned application
+      type: object
+      properties:
+        appInstanceId:
+          $ref: '#/components/schemas/AppInstanceId'
+        edgeResourceName:
+          $ref: '#/components/schemas/EdgeResourceName'
+    EdgeResourceName:
+      description: Edge Resource Name - an identifier for an edge reource in the operator domain 
+      type: string
+ 
+    ErrorInfo:
+      type: object
+      properties:
+        status:
+          type: integer
+          description: HTTP status code returned along with this error response
+        code:
+          type: string
+          description: Code given to this error
+        message:
+          type: string
+          description: Detailed error description
+      required:
+        - status
+        - code
+        - message
+    FileId:
+      type: string
+      format: uuid
+      description: A globally unique identifier assosiated with the image file. Originating OP generates this identifier when file is uploaded over NBI.
+ 
+    MecDetails:
+      type: object
+      required:
+        - geolocation
+      properties:
+        edgeResourceName: 
+          $ref: '#/components/schemas/EdgeResourceName'
+        status:
+          $ref: '#/components/schemas/Status'
+        region:
+            $ref: '#/components/schemas/RegionId'
+ 
+      minItems: 1
+ 
+    OperatingSystem:
+      type: object
+      required:
+        - architecture
+        - family
+        - version
+        - license
+      properties:
+        architecture: 
+          type: string
+          enum:
+            - x86_64
+            - x86
+          example: x86_64  
+        family:
+          type: string
+          enum:
+            - RHEL
+            - UBUNTU
+            - COREOS
+            - WINDOWS
+            - OTROS
+        version:
+          type: string
+          enum:
+            - OS_VERSION_UBUNTU_2204_LTS
+            - OS_VERSION_RHEL_8
+            - OS_VERSION_RHEL_7
+            - OS_VERSION_ DEBIAN_11
+            - OS_VERSION_COREOS_STABLE
+            - OS_MS_WINDOWS_2012_R2
+        license:
+          type: string
+          enum:
+            - OS_LICENSE_TYPE_FREE
+            - OS_LICENSE_TYPE_ON_DEMAND
+ 
+    RegionId:
+      type: string
+      description:  Human readable name of the geografical zone of the MEC. Defined by the OP.
+      enum:
+        - North
+        - South
+        - West
+        - East
```

Comment:
Doesn't seem like this should be an enum, it should be left as a plain string without limiting what values it can be.

Reply:
Agree, updated!